test/test_dist_model_parallel_spawn.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python3
+from __future__ import absolute_import, division, print_function, unicode_literals
```

Comment:
Why do we have two separate files if we only have a `spawn` mode?

Reply:
Let me merge this into one file.